### ISSUE TITLE
Missing Icon

### ISSUE THREAD

**Issue by Dominiquini** [eclipsesource/megit#24]
The application don't have any icon on the taskbar on Windows 10:

![megit_b7efCjgFo2](https://user-images.githubusercontent.com/1180808/197665365-48287881-a977-431d-a076-496ba07a820c.png)

Thanks.

**Comment by planger** [comment]
@codehangen Yes, the project is active. It publishes a release about two to three times a year.

**Comment by Golyshkin** [comment]
For Linux the same issue!

**Comment by fbigozzi-scai** [comment]
I think that this icon would be awesome! https://projects.eclipse.org/sites/default/files/egit.png

**Comment by LouieTian0209** [comment]
> I think that this icon would be awesome! https://projects.eclipse.org/sites/default/files/egit.png

Indeed this icon looks great! It would be nice to see it getting integrated into this project out of the box. But for now we can manually set the icon to be this one.

**Comment by benzjimny** [comment]
@LouieTian0209 Hi, would you please show how to set the icon of megit manually?

**Comment by LouieTian0209** [comment]
> @LouieTian0209 Hi, would you please show how to set the icon of megit manually?

My bad. I was talking about desktop shortcut, didn't realize OP was referring to taskbar icon.

**Comment by eschweighofer** [comment]
Hi, looks like pinning the app to Taskbar shows the icon on taskbar (not on the app window)

<img width="342" height="255" alt="Image" src="https://github.com/user-attachments/assets/49eea9a9-0ac9-4152-a3c8-1ada654e2fd9" />
